data_architect_misc/data_transformer/configs/APAC_Vietnam/step1_transform_raw_sos_investment_data.json

```diff
@@ -48,12 +48,12 @@
       {
         "__function_comment__": "Delete row with value 'Total' from Date column.",
         "function_name": "drop_rows_with_matching_string_values",
-        "function_args": [["DATE", "SECTOR", "BRAND"],[["Total"], ["Total"], ["Total"]]]
+        "function_args": [["DATE", "SECTOR", "BRAND"],["Total","Total","Total"]]
```

Comment:
Why did we removed square brackets in ["Total"]? This is designed so that if we want to delete rows with, for example, 'Total' or 'Sum' in 'Brand' column, we can do something like this:
"function_args": [["DATE", "SECTOR", "BRAND"], [["Total"], ["Total"], ["Total", "Sum"]] ]

Please revert the change to the function if you updated its method signature.

Reply:
This is because the function that we use to delete all these strings, it's not using a list of values because of function uses a str.contain and this funtion no requires a list as parameter